{
  "description": "AuthException is thrown when forgotPassword API call fails",
  "preConditions": {
    "amplify-configuration": "authconfiguration.json",
    "state": "AuthenticationState_SignedIn.json",
    "mockedResponses": [
      {
        "type": "cognito",
        "apiName": "forgotPassword",
        "responseType": "failure",
        "response": {
          "message": "Some cognito error message",
          "codeDeliveryDetails": {
          }
        }
      }
    ]
  },
  "api": {
    "name": "resetPassword",
    "params": {
      "username": "someUsername"
    },
    "options": {
    }
  },
  "validations": [
    {
      "type": "amplify",
      "apiName": "resetPassword",
      "responseType": "failure",
      "response": {
        "message": "Some cognito error message"
      }
    }
  ]
}